tensorflow/tools/test/performance.bzl

```diff
@@ -0,0 +1,55 @@
+# -*- Python -*-
+
+load("//tensorflow:tensorflow.bzl", "tf_py_test")
```

Comment:
Add the tags "local", "regression-test" by default, and you'll have to modify jenkins testing to run:

bazel test $(bazel query "//tensorflow/... except attr(tags, 'local', //tensorflow/...)")

instead of what's currently there:

bazel test //tensorflow/...